dev-infrastructure/modules/mgmt-kv-access.bicep

```diff
@@ -12,24 +12,24 @@ param msiKeyVaultName string
 //
 
 import * as res from 'resource.bicep'
-var clusterServiceMIRef = res.msiRefFromId(clusterServiceMIResourceId)
+var mIRef = res.msiRefFromId(managedIdentityResourceId)
 
-resource clusterServiceMI 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
-  scope: resourceGroup(clusterServiceMIRef.resourceGroup.subscriptionId, clusterServiceMIRef.resourceGroup.name)
-  name: clusterServiceMIRef.name
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
```

Comment:
Just curious, why are we creating Azure resources here rather than as steps in a pipeline in `sdp-pipelines`? Is this just for dev, and then we need to duplicate these for prod/int deployments in `sdp-pipelines`?

Reply:
we have context here and all building blocks ready to create the MI and KV access.